lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -218,6 +218,12 @@ def self.get_name(instance_type, api_name, is_reserved = false)
       'cache.r3.8xlarge' => :ten_gigabit,
       'cache.r3.large' => :moderate,
       'cache.r3.xlarge' => :moderate,
+      'cache.r4.16xlarge' => :twentyfive_gigabit,
```

Comment:
non-blocking: I know it hasn't been enforced but really should sort by size

Reply:
was in (apparent) alphabetical order, so stuck with that pattern. (agree that size order makes more sense.  next time.)